src/query/interpret/eval.hpp

```diff
@@ -313,6 +313,25 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
       }
       return std::nullopt;
     };
+    auto maybe_graph = [this](const auto &graph, const auto &prop_name) -> std::optional<TypedValue> {
+      if (prop_name == "nodes") {
```

Comment:
This is used for `graph.nodes && graph.edges`. Can you plz add some test cases for this?

Reply:
Yes yes, added them.